.github/advanced-issue-labeler.yml

```diff
@@ -67,8 +67,8 @@ policy:
           - name: login
             keys: ['systemd-logind', 'loginctl', 'pam_systemd']
 
-          - name: machined
-            keys: ['systemd-machined', 'machinectl']
+          - name: machine
+            keys: ['systemd-machine', 'machinectl']
```

Comment:
Hmm? Why change this?? The name of the component is systemd-machined

Reply:
Ugh, I completely overlooked this, sorry about that.

Reply:
hey i just made the changes you have told i just made a pr

Reply:
-> https://github.com/systemd/systemd/pull/32378